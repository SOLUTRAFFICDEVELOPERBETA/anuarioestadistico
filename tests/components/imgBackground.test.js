import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import React from 'react';
import ImagBackground from '../../components/imgBackground';

describe('Pruebas en <ImagBackground/>', () => {
    const wrapper = shallow(

        <ImagBackground
            title="Titulo de prueba"
            imgSrc="https://increasify.com.au/wp-content/uploads/2016/08/default-image.png"
            subTitle="Subtitle de prueba"
        />

    )

    test('Debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('debe mostrar la imagen enviada', () => {
        expect(wrapper.find('img').prop('src')).toBe('https://increasify.com.au/wp-content/uploads/2016/08/default-image.png');

        expect(wrapper.find('WithStyles(ForwardRef(Typography))').at(0).text().trim()).toBe('Titulo de prueba')
        expect(wrapper.find('WithStyles(ForwardRef(Typography))').at(1).text().trim()).toBe('Subtitle de prueba')
    });

   
})