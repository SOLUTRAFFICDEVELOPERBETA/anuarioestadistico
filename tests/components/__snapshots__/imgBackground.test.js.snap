// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <ImagBackground/> Debe mostrarse correctamente 1`] = `
<ContainerImagen>
  <img
    alt=""
    src="https://increasify.com.au/wp-content/uploads/2016/08/default-image.png"
  />
  <WithStyles(ForwardRef(Typography))
    align="center"
    className="titulo"
  >
    Titulo de prueba
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))
    align="center"
    className="subTitile"
    component="p"
  >
    Subtitle de prueba
  </WithStyles(ForwardRef(Typography))>
</ContainerImagen>
`;
